src/app/components/Headings/index.jsx

```diff
@@ -13,7 +13,7 @@ export const Headline = styled.h1`
   color: ${C_EBON};
   font-family: ${FF_NEWS_SERIF_MDM};
   margin: 0; // Reset
-  padding: ${GEL_SPACING_QUAD}px 0 ${GEL_SPACING_DBL}px 0;
+  padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_DBL} 0;
 
   // Font styling below is a subset of BBC GEL Typography "Canon"
   font-size: 1.75em;
```

Comment:
Are we not changing `line-height` to use `rem`s too? Currently using `em`s is problematic, due to the relative values. See ems: 
<img width="610" alt="screen shot of scenario 27 with ems" src="https://user-images.githubusercontent.com/3028997/44032937-a7e5ea58-9f00-11e8-8560-ad8b2f2d1f1b.png">

vs rems:
<img width="610" alt="screen shot of scenario 27 with rem" src="https://user-images.githubusercontent.com/3028997/44032945-aba62982-9f00-11e8-9081-3b2bab72058d.png">

on the test article: https://simorgh.test.api.bbci.co.uk/article/scenario-27

Reply:
@sareh yes - let's please pair on this as an additional PR attached to this issue

Reply:
thanks 👍